src/Cli/dotnet/commands/dotnet-test/IPC/ObjectFieldIds.cs

```diff
@@ -85,13 +85,19 @@ internal static class FailedTestResultMessageFieldsId
         public const ushort State = 3;
         public const ushort Duration = 4;
         public const ushort Reason = 5;
-        public const ushort ErrorMessage = 6;
-        public const ushort ErrorStackTrace = 7;
+        public const ushort ExceptionMessageList = 6;
         public const ushort StandardOutput = 8;
```

Comment:
StandardOutput should have the value of 7, ErrorOutput = 8 and SessionUid = 9.

Reply:
will address here: https://github.com/dotnet/sdk/pull/45398